draft-ietf-taps-interface.md

```diff
@@ -2380,11 +2383,27 @@ Default:
 
 This property specifies that a message should be sent and received as a single
 packet without network-layer fragmentation, if possible.
-Attempts to send a message with this property set to a size greater than the
+This only takes effect when used with transports supporting this functionality
```

Comment:
This isn't really about the feature of /transports/ ?????
- Actually this is the transport service when the transport uses a network layer that supports....

Reply:
So, suggest: /This only takes effect when the transport uses a network layer that supports this functionality/